packages/forma-36-website/src/components/Navigation.js

```diff
@@ -161,15 +173,20 @@ MenuListItem.defaultProps = {
   hierarchyLevel: 0,
 };
 
-const MenuList = ({ menuItems, currentPath, hierarchyLevel }) => {
+const MenuList = ({
+  menuItems,
+  currentPath,
+  hierarchyLevel,
+  isHidden = false,
+}) => {
   const activeRef = useRef(null);
   useEffect(() => {
     if (activeRef.current) {
       activeRef.current.scrollIntoView({ block: 'center' });
     }
   }, []);
   return (
-    <ul css={styles.list}>
+    <ul css={[styles.list, isHidden && styles.hidden]}>
```

Comment:
would it not make more sense to solve this via emotion cx?

Reply:
☝️ +1

Reply:
It's essentially the same, only different approach, `cx` would generate the classes to be passed to `className` prop, and the `css` prop is also from emotion.
but we could, but would be kind of merging an array, and passing the merged array to a function that can merge the array if the parameter is an array, and since we won't be doing composition here as we do on the components itself  not sure if needed 🤔

Reply:
@massao you are right - effect wise, it is no difference. but i think we should stick with one dedicated pattern and use the full power of emotion, if we already have committed  to it.

on the other hand, as i am no big fan of css-in-js anyways, i would clearly join in every argument that would move us away from emotion and use css-modules (with sass) only. but i think the mix and match of the patterns makes it harder to understand the applied styles.

Reply:
Sorry i also did not recognise that this is a v3 fix 🙈

Reply:
I think, before emotion, we always preferred to use classNames
since it's the only way to use css modules

but when emotion was introduced, we started to get different ways of doing things just because emotion allow us to do it

the problem that I see is that when a team has too many ways of doing things, it's hard to onboard new developers to the codebase

let's keep like this then until the team decides on how we want to use emotion 🤷